src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,89 @@
+.dashboard-header {
```

Comment:
does the project have a 'main' stylesheet? if so should any of these styles be promoted to that main stylesheet?